alpa/device_mesh.py

```diff
@@ -82,13 +82,27 @@
                                      ["broadcast_specs", "group_name"])
 
 
+class DaemonMoveWorker:
+    """A ray actor that moves local checkpoint into EFS in the background."""
```

Comment:
```suggestion
    """A ray actor that moves local checkpoint to the network file system in the background."""
```

Not everyone uses EFS